Test/dafny0/NestedPatterns.dfy.expect

```diff
@@ -1,9 +1,9 @@
-NestedPatterns.dfy(69,2): Error: member Cons appears in more than one case
-NestedPatterns.dfy(75,2): Error: member  does not exist in datatype List
-NestedPatterns.dfy(76,2): Error: member  does not exist in datatype List
+NestedPatterns.dfy(69,2): Warning: this branch is redundant 
+NestedPatterns.dfy(75,11): Error: member _#Make1 does not exist in type List<_T0>
```

Comment:
I think an attempt to use a 1-tuple should be flagged as an error already in the parser.